### PR TITLE
remove experimental bundle use

### PR DESCRIPTION
**Pull request by aronatkins**
Replaces https://github.com/rstudio/connectapi/pull/189.

### CODE REVIEW

File: man/PositConnect.Rd
Comment (colearendt) [inline man/PositConnect.Rd:1]: Why did this one get deleted? 😮

**Review by colearendt — APPROVED** [review]
LGTM!

File: man/PositConnect.Rd
Comment (aronatkins) [reply man/PositConnect.Rd:1]: These are the functions that we deleted. Most of the file is still there; GitHub is just choosing to hide the generated doc files from you.

Comment (colearendt) [reply man/PositConnect.Rd:1]: 🤦 Wow, that's a wild miss. Sorry about that! Not sure how/why I interpreted that as the file getting deleted

**Pull request merged**